provider/bitmovin/bitmovin.go

```diff
@@ -987,6 +992,56 @@ func (p *bitmovinProvider) addOutputStatusInfo(job *db.Job, status *provider.Job
 	return nil
 }
 
+func (p *bitmovinProvider) addOutputFilesInfo(job *db.Job, status *provider.JobStatus) error {
+	// TODO: we only have info for MP4. Need to add for webm and mov.
+
+	muxings, err := p.listMP4Muxing(job.ProviderJobID)
+	if err != nil {
+		return err
+	}
+
+	var files []provider.OutputFile
+	for _, muxing := range muxings {
+		encodingS := services.NewEncodingService(p.client)
+		resp, err := encodingS.RetrieveMP4MuxingInformation(job.ProviderJobID, stringValue(muxing.ID))
+		if err != nil {
+			return err
+		}
+
+		info := resp.Data.Result
+		if len(info.VideoTracks) > 0 {
```

Comment:
Should it return error if `len(VideoTracks) == 0`?

Reply:
Yeah, since this is a required info, let's fail in case it's missing.